### DISCUSSION TITLE
Using m-ld-js@^0.9

### DISCUSSION THREAD

**Discussion by ghost** [m-ld/m-ld-spec#92] [How-to] [answered]
Hello all,

I am trying to work with the m-ld-js library and the current version is 0.9, and unfortunately the examples are version ^0.8, and there appear to be breaking changes. Does anyone have an example similar to the node one using m-ld@^0.9, or will the examples be updated soon?

Thank you in advance!

**Comment by gsvarovsky** [comment, marked as answer]
Edit: [Javascript engine v0.9](https://www.npmjs.com/package/@m-ld/m-ld/v/0.9.0) has been released! Docs are available at https://js.m-ld.org/

<details>
  <summary>Old answer</summary>
Hiya! ^0.9 is a pre-release at the moment because the new features aren't fully documented yet. However, it's almost ready to go, and the changes to existing features are minor.

The current (incomplete) docs are at edge.js.m-ld.org

I've created a corresponding `edge` branch on the m-ld-nodejs-starter project and fixed it up for ^0.9, see [PR diff](https://github.com/m-ld/m-ld-nodejs-starter/pull/1/files)

I've also tried the web-starter project and discovered that it exposes a bug in the pre-release! I'll get onto that right away.
_Update: fixed, also on [edge](https://github.com/m-ld/m-ld-web-starter/tree/edge)_

I think maybe we should not be labelling the pre-releases of m-ld-js as `latest` on npm. Hmm...
</details>

**Reply by ghost** [reply]
Okay, thanks! 

I'll look for the 0.9 release and just roll back to 0.8 for now.
